rules/possible-errors.js

```diff
@@ -0,0 +1,34 @@
+// Possible Errors http://eslint.org/docs/rules/#possible-errors
+
+module.exports = {
+	"rules": {
+		"comma-dangle": [ 2, "never" ], // JSCS disallow or enforce trailing commas
+		"no-cond-assign": [ 2, "always" ], // JSHINT disallow assignment in conditional expressions
+		"no-console": 2, // disallow use of console in the node environment
+		"no-constant-condition": 2, // disallow use of constant expressions in conditions
+		"no-control-regex": 2, // disallow control characters in regular expressions
+		"no-debugger": 2, // JSHINT disallow use of debugger
+		"no-dupe-args": 2, // disallow duplicate arguments in functions
+		"no-dupe-keys": 2, // disallow duplicate keys when creating object literals
+		"no-duplicate-case": 2, // disallow a duplicate case label
+		"no-empty-character-class": 2, // disallow the use of empty character classes in regular expressions
+		"no-empty": 2, // JSHINT disallow empty statements
+		"no-ex-assign": 2, // disallow assigning to the exception in a catch block
+		"no-extra-boolean-cast": 2, // disallow double-negation boolean casts in a boolean context
+		"no-extra-parens": 1, // disallow unnecessary parentheses
```

Comment:
Discuss, do we want this to be ignored (0) or to be an error (2)?

http://eslint.org/docs/rules/no-extra-parens.html